src/calendars/parsing/ics.ts

```diff
@@ -1,17 +1,23 @@
 import ical from "ical.js";
 import { OFCEvent, validateEvent } from "../../types";
-import { DateTime } from "luxon";
+import { DateTime, DateTimeJSOptions } from "luxon";
```

Comment:
why not use an `import type` here?
```suggestion
import { DateTime, type DateTimeJSOptions } from "luxon";
```